library/edi.inc

```diff
@@ -581,171 +471,162 @@ function show_elig($res, $X12info, $segTer, $compEleSep)
+        "WHERE insd.pid = ? AND eligv.eligibility_check_date = " .
+        "(SELECT Max(eligibility_verification.eligibility_check_date) FROM eligibility_verification ".
+        "WHERE eligibility_verification.insurance_id = eligv.insurance_id)";
+    $result = sqlStatement($query, array($pid));
+
+    $showString = "<div class='tab-content pre-scrollable' style='width:695px;overflow-x: hidden;'><div class='row'>";
+    $col = 1;
+    $title = 1;
+    while ($benefit = sqlFetchArray($result)) {
+        if ($title) {
+            $title = 0;
+            $showString .= "<div class='col col-sm-12'>\n";
+            $showString .= "<b>" . xlt('Insurance Provider') . ":</b> " . (!empty($benefit['name']) ? text($benefit['name']) : xlt('n/a')) . "<br/>\n";
+            $showString .= "<b>" . xlt('Verified On') . ":</b> " . $benefit['verificationDate'] . "<br/><br/>\n";
+            $showString .= "</div><br/>\n";
+        }
+        $benefit['start_date'] = strpos($benefit['start_date'], "0000") === false ? $benefit['start_date'] : '';
+        $benefit['end_date'] = strpos($benefit['end_date'], "0000") === false ? $benefit['end_date'] : '';
+
+        $showString .= "\n<div class='col col-sm-6'>\n";
+        $showString .= !empty($benefit['benefit_type']) ? "<b>" . xlt('Benefit Type') . ":</b> " .  $benefit['benefit_type'] . "<br />\n" : '';
+        $showString .= !empty($benefit['start_date']) ? "<b>" . xlt('Start Date') . ":</b> " . date("d/m/Y", strtotime($benefit['start_date'])) . "<br />\n" : '';
+        $showString .= !empty($benefit['end_date']) ? "<b>" . xlt('End Date') . ":</b> " . date("d/m/Y", strtotime($benefit['end_date'])) . "<br />\n" : '';
+        $showString .= !empty($benefit['coverage_level']) ? "<b>" . xlt('Coverage Level') . ":</b> " .$benefit['coverage_level'] . "<br />\n" : '';
+        $showString .= !empty($benefit['coverage_type']) ? "<b>" . xlt('Coverage Type') . ":</b> " . $benefit['coverage_type'] . "<br />\n" : '';
+        $showString .= !empty($benefit['plan_type']) ? "<b>" . xlt('Plan Type') . ":</b> " .  $benefit['plan_type'] . "<br />\n" : '';
+        $showString .= !empty($benefit['plan_desc']) ? "<b>" . xlt('Plan Description') . ":</b> " .  $benefit['plan_desc'] . "<br />\n" : '';
+        $showString .= !empty($benefit['coverage_period']) ? "<b>" . xlt('Coverage Period') . ":</b> " .$benefit['coverage_period'] . "<br />\n" : '';
+        $showString .= !empty($benefit['amount']) ? "<b>" . xlt('Amount') . ":</b> " . $benefit['amount'] . "<br />\n" : '';
+        $showString .= !empty($benefit['percent']) ? "<b>" . xlt('Percentage') . ":</b> " . $benefit['percent'] . "<br />\n" : '';
+        $showString .= !empty($benefit['network_ind']) ? "<b>" . xlt('Network Indicator') . ":</b> " . $benefit['network_ind'] . "<br />\n" : '';
+        $showString .= !empty($benefit['message']) ? "<b>" . xlt('Message') . ":</b> " . $benefit['message'] . "<br />\n" : '';
```

Comment:
text() around all the $benefit['*'] outputs above

Reply:
oops going to html.